nautobot/core/templates/generic/object_list.html

```diff
@@ -147,7 +121,6 @@
     {% endif %}
 </div>
 <h1>{% block title %}
-    {{ title }}
```

Comment:
Just a note. Removing the title from here also causes the title tag to be incomplete in the browser tab. It is rendered as "- Nautobot" right now
![Screenshot 2025-04-02 at 10 14 00 AM](https://github.com/user-attachments/assets/bf0f4f79-5e84-4fb4-a586-c3856e022b01)